xlators/features/shard/src/shard.h

```diff
@@ -349,6 +349,7 @@ typedef struct shard_inode_ctx {
     inode_t *inode;
     int fsync_count;
     inode_t *base_inode;
+    gf_boolean_t i_ctx_refresh_protect;
```

Comment:
nit: while the structure is not aligned properly, this one is just making it worse. Attach it to one of the previous booleans, at least.

Reply:
My bad. will make this change in next patch set